modules/light-clients/08-wasm/types/client_message.go

```diff
@@ -6,15 +6,15 @@ import (
 	"github.com/cosmos/ibc-go/v7/modules/core/exported"
 )
 
-var _ exported.ClientMessage = &Header{}
+var _ exported.ClientMessage = &ClientMessage{}
 
-// ClientType defines that the Header is a Wasm client consensus algorithm
-func (h Header) ClientType() string {
+// ClientType defines that the client message is a Wasm client consensus algorithm
+func (h ClientMessage) ClientType() string {
```

Comment:
we can change `h` -> `c`/`cm`?

```suggestion
func (c ClientMessage) ClientType() string {
```